pixi.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+authors = ["Matthew Feickert <matthew.feickert@cern.ch>"]
```

Comment:
Is there a Scientific Python org email that we can use here instead? `pixi` generates this information from your Git credentials automatically during `pixi init`.

Reply:
Are there alias email addresses for e.g. a discorse channel? That case it would be a fully transparent address. 

Also, the docs says that authors is optional, so maybe leaving it empty or as `Scientific Python`, as in the Licence, without an email address could work, too.

Reply:
We can make a discourse group, and attach a x@discuss.scientific-python.org address to it, or I can make an alias for x@scientific-python.org and redirect it wherever. We don't have the concept of forward-email-into-discord atm.